components/widgets/resources/src/main/resources/resources/uicomponents/widgets/extraGeneVariantData.js

```diff
@@ -353,6 +353,18 @@ var ExtraGeneVariantData = (function (ExtraGeneVariantData) {
       newMoreInfoRow.insert(varMoreInfoInner);
       variantFooter.insert({before : newMoreInfoRow});
 
+      // restrict only numerical input for start_position and end_position variant fiellds
+      newMoreInfoRow.select('input:[id$=_start_position]', 'input:[id$=_end_position]').each( function(input) {
+        input.observe('keyup', function(event){
+          this.value=this.value.replace(/[^\d]/,'');
+        });
+      });
+
+      // get the last value of reference_genome of variants related to current gene symbol to create a new variant row with this value
```

Comment:
That is true, however IMO it's quite unlikely users would actually jump from the 2nd gene back to the first one. We should do our best make it easy for users follow the steps in the order they'd be 95% of the time, which is transferring data from a report to the electronic record top to bottom.